### PR TITLE
stub after deprecation

### PR DESCRIPTION
**Pull request by ianbjacobs**
The following tasks have been completed:

 * [x] Confirmed there are no ReSpec errors/warnings.

To do:

- [ ] update PR API, as it references this spec.
- [x] update the README.
- [ ] buy @ianbjacobs a beer.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/payment-method-basic-card/pull/90.html" title="Last updated on Aug 26, 2021, 12:06 PM UTC (0285642)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/payment-method-basic-card/90/aa90049...0285642.html" title="Last updated on Aug 26, 2021, 12:06 PM UTC (0285642)">Diff</a>

### CODE REVIEW

**Comment by ianbjacobs** [comment]
@marcoscaceres, Please have a look. I will update the pull request with the link to the WG decision (anticipated 26 August).

File: index.html [hunk 1]
```diff
@@ -61,709 +61,14 @@
...
-        make use of this specification.
-      </p>
-      <p>
-        In particular, merchants need to treat the values of any <a>details</a>
-        with the same scrutiny that they would apply to a [[HTML]] `input`
-        element, by, for example, sanitizing all the members of a
-        {{BasicCardResponse}} before rendering them anywhere.
+        <strong>The <a href="https://www.w3.org/Payments/WG/">Web Payments
+        Working Group</a> has deprecated this specification. See the <a href=
+        "@@">Working Group decision</a> for more information.</strong>
```
Comment (marcoscaceres) [inline index.html]:
```suggestion
        <strong>The <a href="https://www.w3.org/Payments/WG/">Web Payments
        Working Group</a> has discontinued work on this document. See the <a href=
        "@@">Working Group decision</a> for more information.</strong>
```

File: index.html [hunk 2]
```diff
@@ -61,709 +61,14 @@
...
       </p>
       <p>
-        Depending on jurisdiction, users of this specification (implementers,
-        merchants, payment processors, etc.) can be subject to <dfn data-abbr=
-        "PCI DSS">Payment Card Industry Data Security Standard Compliance</dfn>
-        or other regulations. Discussion of those considerations are outside
-        the scope of this document.
+        <strong>The remainder of this status section is boilerplate required of
```
Comment (marcoscaceres) [inline index.html]: Not sure if it's worth adding this?

**Review by marcoscaceres — COMMENTED** [review]
Few suggestions... note that this has ramifications on Payment Request, as we use Basic Card in a bunch of examples. Would you mind sending a pull request to the PR API too?

**Comment by marcoscaceres** [comment]
PS: don't worry about the ReSpec config... I'll handle that separately. It's also changed by the auto-pub script.

**Comment by marcoscaceres** [comment]
@ianbjacobs, could you please also update the README? 

After this goes in and we publish we can archive the repo.

**Comment by marcoscaceres** [comment]
Added a todo to the top of the pull request.

**Comment by ianbjacobs** [comment]
See also PR for Payment Request to remove references:
 https://github.com/w3c/payment-request/pull/963

**Comment by ianbjacobs** [comment]
I unblocked this now that we've announced the decision. There are two other pull requests associated with this one:
https://github.com/w3c/payment-request/pull/963
and
https://github.com/w3c/payment-method-id/pull/66

**Review by marcoscaceres — APPROVED** [review]
So long, 3 years of my life 👋😭

**Pull request merged**